### ISSUE TITLE
wiring diagram

### ISSUE THREAD

**Issue by avfoto** [tfyoung/esphome-junctek_kgf#20]
Forgive me if I'm wrong, but to connect the KG140F to my Esp32, is it correct that I have to connect the two cables (A and B) that come out of the RS485 port of the unit, to the corresponding GPIO ports of my Esp32?
Thanks for the work.

![Captura de pantalla 2023-12-30 a las 21 02 04](https://github.com/tfyoung/esphome-junctek_kgf/assets/5754971/525ee6b4-4e03-4c14-9055-07c6b159dc93)

**Comment by gropi75** [comment]
Not really. You need also an RS485 bus driver. Like [here](https://embeddedthere.com/how-to-interface-esp32-with-rs485-modbus-sensors-with-example-code/).

**Comment by avfoto** [comment]
Ah ok, many thanks.
